charts/seed-controlplane/charts/kube-apiserver/templates/_helpers.tpl

```diff
@@ -90,3 +90,24 @@ rules:
 - level: None
 {{- end -}}
 {{- end -}}
+
+{{- define "kube-apiserver.serviceAccountConfig" -}}
+{{- if .Values.serviceAccountConfig }}
+{{- if .Values.serviceAccountConfig.issuer }}
+- --service-account-issuer={{ .Values.serviceAccountConfig.issuer }}
+{{- end }}
+{{- if .Values.serviceAccountConfig.signingKeyFile }}
+- --service-account-signing-key-file={{ .Values.serviceAccountConfig.signingKeyFile }}
```

Comment:
This must be a path to a file. You probably should create a `Secret` that contains `{{ .Values.serviceAccountConfig.signingKeyFile }}`. The path should not be configurable by the user but determined by us.